database/mssql/scripts/dbscripts/PSP_PIMS_S17_00/Alter Up/01_PSP_PIMS_S16_01_Alter_Up.sql

```diff
@@ -0,0 +1,940 @@
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_PERSON
+PRINT N'Alter table dbo.PIMS_PERSON'
+GO
+ALTER TABLE [dbo].[PIMS_PERSON]
+	ADD [ADDRESS_COMMENT] nvarchar(2000) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_PERSON_HIST
+PRINT N'Alter table dbo.PIMS_PERSON_HIST'
+GO
+ALTER TABLE [dbo].[PIMS_PERSON_HIST]
+	ADD [ADDRESS_COMMENT] nvarchar(2000) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_INSURANCE
+PRINT N'Alter table dbo.PIMS_INSURANCE'
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE] ALTER COLUMN [COVERAGE_LIMIT] money NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE] ALTER COLUMN [EXPIRY_DATE] date NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE]
+	ADD [IS_INSURANCE_IN_PLACE] bit NULL CONSTRAINT [INSRNC_IS_INSURANCE_IN_PLACE_DEF] DEFAULT (CONVERT([bit],(0)))
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE]
+	DROP COLUMN [INSURER_ORG_ID], [INSURER_CONTACT_ID], [MOTI_RISK_MGMT_CONTACT_ID], [BCTFA_RISK_MGMT_CONTACT_ID], [INSURANCE_PAYEE_TYPE_CODE], [INSURED_VALUE], [START_DATE], [RISK ASSESSMENT_COMPLETED_DATE]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Create unique constraint dbo.INSRNC_LEASE_INSURANCE_TYPE_TUC
+PRINT N'Create unique constraint dbo.INSRNC_LEASE_INSURANCE_TYPE_TUC'
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE]
+	ADD CONSTRAINT [INSRNC_LEASE_INSURANCE_TYPE_TUC]
+	UNIQUE ([INSURANCE_TYPE_CODE], [LEASE_ID]) 
+	WITH (
+		DATA_COMPRESSION = NONE
+	) ON [PRIMARY]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_INSURANCE_HIST
+PRINT N'Alter table dbo.PIMS_INSURANCE_HIST'
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE_HIST] ALTER COLUMN [COVERAGE_LIMIT] money NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE_HIST] ALTER COLUMN [EXPIRY_DATE] date NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+EXEC dbo.sp_rename N'dbo.PIMS_INSURANCE_HIST.INSURER_ORG_ID' , N'IS_INSURANCE_IN_PLACE', 'COLUMN'
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE_HIST] ALTER COLUMN [IS_INSURANCE_IN_PLACE] bit NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE_HIST]
+	DROP COLUMN [INSURER_CONTACT_ID], [MOTI_RISK_MGMT_CONTACT_ID], [BCTFA_RISK_MGMT_CONTACT_ID], [INSURANCE_PAYEE_TYPE_CODE], [INSURED_VALUE], [START_DATE], [RISK ASSESSMENT_COMPLETED_DATE]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_LEASE_PAYMENT
+PRINT N'Alter table dbo.PIMS_LEASE_PAYMENT'
+GO
+ALTER TABLE [dbo].[PIMS_LEASE_PAYMENT] ALTER COLUMN [PAYMENT_AMOUNT_PST] money NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_LEASE_PAYMENT_HIST
+PRINT N'Alter table dbo.PIMS_LEASE_PAYMENT_HIST'
+GO
+ALTER TABLE [dbo].[PIMS_LEASE_PAYMENT_HIST] ALTER COLUMN [PAYMENT_AMOUNT_PST] money NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Create view dbo.PIMS_CONTACT_MGR_VW
+PRINT N'Create view dbo.PIMS_CONTACT_MGR_VW'
```

Comment:
@dfilteau why is the view being recreated here? were there any changes?

Reply:
This is the file I asked @dfilteau to remove on his previous S16_03 PR. It seems like it got saved in the wrong folder
Could you please confirm @dfilteau that it can be safely removed?

Reply:
I see now that this file contains the whole Alter Up statements. Was thinking it only had the contact manager view in it. 
So no, it cannot be deleted but perhaps this view can be removed from the alter up?

The only change to this view would be using the new address type code MAILING instead of the old MAILADDR one. But I was under the impression that the change already was applied in S16 or am I mistaken @dfilteau ?

Reply:
It was applied in S16 via a conversation with @devinleighsmith and in another conversation with Devin, these small changes were intended to be a record of the change that was made "by hand" by the developers.